habit/src/actions/login.js

```diff
@@ -0,0 +1,20 @@
+import Login from "../containers/login/LoginScreen";
+
+export const login = (data) => {
+	let { email, password } = data;
+	email = email.toLowerCase();
+	return (dispatch) => {
+		return fetch('/login', {
+			method: 'POST',
+			headers: {  //TODO AYUSHI change them according to the API
+				Accept: 'application/json',
+				'Content-Type': 'application/json',
+			},
+			body: JSON.stringify(email),
+		})
+		.then((response) => dispatch({
+			type: 'Login',
```

Comment:
I believe we'll receive the JWT and the user object here, which we should save in our redux store